### ISSUE TITLE
[Feature Request] Toggle Folding

### ISSUE THREAD

**Issue by tmueller** [jamalsenouci/sublimetext-syntaxfold#17]
Hi,

I am using your Plugin and it works really well. It would be nice to have a toggle folding command, so one has to rememeber only a single keyboard shortcut.

**Comment by jamalsenouci** [comment]
I agree. This should be an easy one as well, unfortunately I haven't got any time to work on enhancements for this plugin

**Comment by Armfoot** [comment]
Hi Jamal, thank you for this plugin. It is one of the most useful ones I have installed (I actually found it due to my usage of Visual Studio regions).

It was easy to modify as you wrote, I just needed to add a condition in your fold command based on:

> self.view.is_folded(self.view.sel()[0])

I renamed the `FoldCurrentCommand` to `ToggleFoldCurrentCommand` and removed the unfold command.
I also left the (un)fold all commands untouched, since people may still use them without wanting them all to be toggled.
